### PR TITLE
[CORE24-343]: redirect to list after submit

### PR DESCRIPTION
**Pull request by internetti**
Jira ticket: JIRA-CORE24-343

## What have you implemented?

redirect to list view after entity has been submitted

### CODE REVIEW

**Review by dfigueroa87 — APPROVED** [review]
Not sure if it was part of the requirements, but I think we should show some sort of success message after creation

File: libs/prm-components/src/lib/pages/EntityListPage/EntityListPage.component.tsx [hunk 1]
```diff
@@ -45,12 +47,19 @@ export const EntityListPage: React.FC = () => {
     data,
   } = useEntityListPage(pagination);
 
+  const [searchParams, setSearchParams] = useSearchParams();
```
Comment (dfigueroa87) [inline libs/prm-components/src/lib/pages/EntityListPage/EntityListPage.component.tsx:50]: Why search? What does search has to do with this?

Comment (internetti) [reply libs/prm-components/src/lib/pages/EntityListPage/EntityListPage.component.tsx:50]: I sent a success flag as a query parameter in the url, which for react-router-dom is a search parameter

**Review by dfigueroa87 — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**